### PR TITLE
FixUp Copyright

### PR DESCRIPTION
**Pull request by Bukhtawar**
### Description
[Describe what this change achieves]
 
### Issues Resolved
https://github.com/opensearch-project/asynchronous-search/issues/28
### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -1,9 +1,2 @@
 OpenSearch
-Copyright 2021 OpenSearch Contributors
-
-This product includes software developed by
```
Comment (dblock) [inline NOTICE:4]: Please confirm that asynchronous-search *does not* contain code from Elasticsearch or the Apache Software Foundation, and that it was just a bad copy-paste?

**Comment by dblock** [comment]
Needs a rebase/passing CI?

File: NOTICE (hunk 1 above)
Comment (Bukhtawar) [reply NOTICE:4]: We have dependencies on core and ASF however this plugin is solely developed from scratch by the Opendistro community members

Comment (dblock) [reply NOTICE:4]: Dependencies != code copied, good

**Pull request closed without merging**